cmake/modules/DefineCompilerFlags.cmake

```diff
@@ -56,6 +56,13 @@ if (${CMAKE_C_COMPILER_ID} MATCHES "(GNU|Clang)")
     endif()
 endif (${CMAKE_C_COMPILER_ID} MATCHES "(GNU|Clang)")
 
+# Colored compiler output when using Ninja
```

Comment:
why do you say it is linked to ninja ?

Reply:
I am not sure the comment is needed here, the code behind it looks easy to read

Reply:
Because it is colored when using the Makefile generator. I'm not sure why CMake does not add this flag to Ninja by default.

Reply:
But I get your point. It's adding the flag also to Makefile builds.

Reply:
but you kept the comment ?

Reply:
I do not think the comment is worth keeping especially because you are not checking the cmake generator to be only ninja and the options added to the compiler command lines are easy to read

Reply:
I removed it :) We have the information in the commit message